packages/main/src/ToggleButton.ts

```diff
@@ -50,34 +35,42 @@ const metadata = {
  * @tagname ui5-toggle-button
  * @public
  */
+@customElement("ui5-toggle-button")
 class ToggleButton extends Button {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Determines whether the component is displayed as pressed.
+	 *
+	 * @type {boolean}
+	 * @defaultvalue false
+	 * @name sap.ui.webc.main.ToggleButton.prototype.pressed
+	 * @public
+	 */
+	@property({ type: Boolean })
+	pressed!: boolean;
 
 	static get template() {
 		return ToggleButtonTemplate;
 	}
 
-	static get styles() {
+	static get styles(): ComponentStylesData {
 		return [Button.styles, toggleBtnCss];
 	}
 
 	_onclick() {
 		this.pressed = !this.pressed;
 
 		if (isSafari()) {
-			this.getDomRef().focus();
+			this.getDomRef()?.focus();
```

Comment:
use ! instead of ?. You can't have click method if domref doesn't exist.

Reply:
correct